src/dune_config/dune_config.mli

```diff
@@ -121,6 +124,7 @@ val equal : t -> t -> bool
 
 val for_scheduler :
      t
+  -> ?watch_exclusions:string list
```

Comment:
Please document that the default value is `standard_watch_exclusions`.

Reply:
i.e. a comment:
```
(** [for_schedular] t ?watch_excludsions ... does this and this. The default value of watch_ is this. *)

```

Reply:
done